pacman/model/partitioner_interfaces/abstract_splitter_partitioner.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020-2021 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from six import add_metaclass
+from spinn_utilities.abstract_base import AbstractBase, abstractmethod
+
+
+@add_metaclass(AbstractBase)
+class AbstractSplitterPartitioner(object):
```

Comment:
I dont see the need for an abstract here,

Does it really make sense to think there will be multiple implementations.
And no a Spynnaker specific ones doe snot count as I hope that will be a subclass of the pacman one.

Reply:
it was there to remind me that when building the fec / spy parittioners that that method had to be overrridable. else logic for overriding the spyn one was going to fail epically. 

now...... it at least enforces that seperation..........stops us from thinking we could optmise it out at a later date and wrecking the spy tie in.

the other solution is to have some function hand over. which i dislike more

Reply:
@Christian-B on skype undertsands now. so resolving